torch/_jit_internal.py

```diff
@@ -783,7 +788,7 @@ def _get_overloaded_methods(method, mod_class):
 
 
 def is_tuple(ann) -> bool:
-    if ann is Tuple:
+    if ann is Tuple or ann is tuple:
```

Comment:
There seem to be some action-at-a-distance test failures happening. guessing but maybe this would work?

```
if ann is Tuple or getattr(ann, '__origin__', None) is tuple:
```

(Note i'm not 100% sure why the change here is needed. where are these type annotations that are not the `typing` classes coming from?

Reply:
Turns out that "fixing" code without reading the line directly below it is Not Good™. Uh, I've got half of "move fast, break things" down?